### DISCUSSION TITLE
Translation help

### DISCUSSION THREAD

**Discussion by ghost** [Floorp-Projects/Floorp#332] [Q&A] [answered]
I want to translate Floorp into my own language (tr-TR), but I am not sure which parts to translate.

Only the parts after the equal sign, or also the titles and the words separated by dashes?

I assume the first option.

**Comment by Ryosuke-Asano** [comment, marked as answer]
https://github.com/Floorp-Projects/Floorp/raw/ESR115/floorp/browser/locales/en-US/floorp.ftl

Plz translate this file.  After that, plz send me a file.

**Reply by Ryosuke-Asano** [reply]
TR is not supported now. But I will add it if you translate it.

**Reply by ghost** [reply]
I understood that part since I read the readme. Pls reread my post. Am I supposed to translate everything there? Even things like ``.label``  and ``enable-multitab``

If there is already translated .ftl for a language other than english, that would be helpful for me

**Reply by Ryosuke-Asano** [reply]
I didn't read it properly. sorry.

yes.  Translate everything after these "="

https://github.com/Floorp-Projects/l10n-central/blob/11.0.0/l10n-central/ja/browser/browser/floorp.ftl
